cEP-0005.md

```diff
@@ -85,6 +85,18 @@ files = **.(c|h)
 ignore_aspects = redundancy  # Exclusive
 ```
 
+aspect name could be writen by its fully qualified name
```

Comment:
Per https://github.com/coala/cEPs/issues/69#issuecomment-293317399 and prior comments, the cEP should not be forced to use lowercase `aspects` instead of using proper grammar, which is capitalised "Aspects" at the beginning of a sentence.

So, either use "Aspects" here, or adjust the sentence so that the word "aspects" is not the start of the sentence.